testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -1,1313 +0,0 @@
-    let val_config_resource_response = client
-        .get_resource::<aptos_types::validator_config::ValidatorConfig>(
-            validator_account,
-            std::str::from_utf8(&aptos_types::validator_config::ValidatorConfig::resource_path())
-                .unwrap(),
-        )
-        .await
-        .unwrap();
-    let _val_config_resource = val_config_resource_response.inner().clone();
-}
-
-/// Launches a validator swarm of a specified size, connects an operational
-/// tool to the first node and fetches that node's secure backend.
-pub async fn launch_swarm_with_op_tool_and_backend(
-    num_nodes: usize,
-) -> (LocalSwarm, OperationalTool, SecureBackend, Storage) {
-    let swarm = new_local_swarm_with_aptos(num_nodes).await;
-    let chain_id = swarm.chain_id();
-    let validator = swarm.validators().next().unwrap();
-
-    // Connect the operator tool to the node's JSON RPC API
-    let op_tool = OperationalTool::new(validator.rest_api_endpoint().to_string(), chain_id);
-
-    // Load validator's on disk storage
-    let backend = load_validators_backend_storage(validator);
-    let storage: Storage = (&backend).try_into().unwrap();
-
-    (swarm, op_tool, backend, storage)
-}
-
-/// Writes a given key to file using a specified file writer and test environment.
-fn write_key_to_file(
-    key: &Ed25519PublicKey,
-    node: &LocalNode,
-    file_writer: fn(&Ed25519PublicKey, PathBuf),
-) -> PathBuf {
-    let node_config_path = node.config_path();
-    let file_path = node_config_path.with_file_name("KEY_FILE");
-    file_writer(key, file_path.clone());
-    file_path
-}
-
-async fn wait_for_account_sequence_number(
-    client: &RestClient,
-    address: AccountAddress,
-    seq: u64,
-) -> Result<()> {
-    const DEFAULT_WAIT_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(120);
-
-    let start = std::time::Instant::now();
-    while start.elapsed() < DEFAULT_WAIT_TIMEOUT {
-        let txns = client
-            .get_account_transactions(address, Some(seq), Some(1))
-            .await?
-            .into_inner();
-        if txns.len() == 1 {
-            return Ok(());
-        }
-        tokio::time::sleep(std::time::Duration::from_millis(10)).await;
-    }
-    bail!(
-        "wait for account(address={}) transaction(seq={}) timeout",
-        address,
-        seq
-    )
-}
-
-pub async fn wait_for_transaction_on_all_nodes(
-    swarm: &LocalSwarm,
-    account: AccountAddress,
-    sequence_number: u64,
-) {
-    for validator in swarm.validators() {
-        let client = validator.rest_client();
-        wait_for_account_sequence_number(&client, account, sequence_number)
-            .await
-            .unwrap();
-    }
-}
-
-async fn generate_x25519_key(path: &Path) -> x25519::PrivateKey {
-    let (priv_key, _pub_key) =
-        GenerateKey::generate_x25519(aptos::common::types::EncodingType::Hex, path)
-            .await
-            .unwrap();
-    priv_key
-}
```

Comment:
Seems like we are deleting a lot of tests here... Why? Are we planning to add them back later on? Who's going to remember to do that?

Reply:
all the deleted tests are ignored today, we're rewriting those operational tooling and need to rewrite those tests one day